src/utils/test-utils.ts

```diff
@@ -89,3 +105,74 @@ export const awaitUntilIndexed = async (client: HttpClient | Index, timeoutMilli
 
   throw new Error(`Indexing is not completed in ${timeoutMillis} ms.`);
 };
+
+export const populateSparseIndex = async (index: Index, namespace: string) => {
```

Comment:
Can we maybe parametrize this function with dimension count etc. and randomize the vectors? So that we can use it in multiple places

Reply:
can we do this at a later date when we need it, I don't want to refactor all tests at this stage :D